app/views/users/_created.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="p-4">
+
+<% if events_created_by(@user).empty? %>
+  <%= "There's not events created by #{@user.name}" %>
+<% else %>
+<% events_created_by(@user).each do |event| %>
```

Comment:
You should use a variable here instead. You can create it in the user show controller.